jsonslice.go

```diff
@@ -1111,6 +1116,22 @@ func doFunc(input []byte, nod *tNode) ([]byte, error) {
 		} else {
 			return nil, errInvalidLengthUsage
 		}
+	} else if bytes.Equal(word("now"), nod.Keys[0]) || bytes.Equal(word("nowRFC3339"), nod.Keys[0]) {
+		var val interface{}
+		err = json.Unmarshal(input, &val)
+		if err != nil {
+			return nil, errInvalidNowUsage
+		}
+		var t string
+		if bytes.Equal(word("nowRFC3339"), nod.Keys[0]) {
+			//parse to RFC3339
+			t = time.Now().Format(time.RFC3339)
+		} else {
+			//parse to ISO8601
+			t = time.Now().Format("2006-01-02T15:04:05.000Z")
+		}
+		result, _ := json.Marshal(t)
+		return result, nil
```

Comment:
Since passing a parameter doesn't work, can this instead act on the preceding data?
Something like `$.[timestamp].RFC3339()`? That way it can be used on the previous operator `$.now()` and also any other timestamp provided to it

Reply:
that's fair. i've pushed an interpretation of this. let me know what you think